Code/Legacy/CryCommon/Cry_Quat.h

```diff
@@ -395,6 +395,30 @@ struct Quat_tpl
         return q;
     }
 
+    // creatr rotation quatation base on ZYX axis
+    ILINE void SetRotationZYX(const Ang3_tpl<F>& a)
```

Comment:
update: If we could look at AZ::Quaternion and see if this functionality is already supported could we use that instead (we are long-term planning to deprecate and remove CryMath so adding more functionality should ideally be avoided if possible

Reply:
We could do this as a follow-up PR if needed, but might be nice to do a pass over Track View (with tests) to update the use of the math library (this is of course a large change though, so might need some thought/planning).